web/cobrands/fixmystreet.com/fmsforcouncils.scss

```diff
@@ -204,6 +225,10 @@ $fms-green: #62b356;
     margin-top: 18px;
   }
 
+  .councils-features__screenshot {
```

Comment:
That was there as I planned to have code ready for screenshots to be added, didn't get round to it in the end though, will remove